### ISSUE TITLE
git-pw: UnicodeEncodeError: ascii' codec can't encode character

### ISSUE THREAD

**Issue by borkmann** [getpatchwork/git-pw#52]
Seen with git-pw downloading the mbox by series id:

https://patchwork.ozlabs.org/patch/1169842/

```
$ git pw --version
git-pw, version 1.5.1
```

**Comment by borkmann** [comment]
One more for the series id of below two:
https://patchwork.ozlabs.org/patch/1170615/
https://patchwork.ozlabs.org/patch/1170616/

```
$ git pw series download 133745 mbox.i
Server version missing
You should provide the server version in the URL configured via git-config or --server
This will be required in git-pw 2.0
Traceback (most recent call last):
  File "/usr/bin/git-pw", line 10, in <module>
    sys.exit(cli())
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/git_pw/series.py", line 55, in download_cmd
    output.write(api.get(series['mbox']).text)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 4418-4420: ordinal not in range(128)
```

**Comment by borkmann** [comment]
By the way, download of the series mbox via web UI (e.g. `https://patchwork.ozlabs.org/series/133745/mbox/`) works fine and git am takes it.

**Comment by stephenfin** [comment]
This was fixed in 1.7.0 (commit ee93afcd). You need to update :smile:

**Comment by borkmann** [comment]
Ah, great! Will do, thanks!
